content/en/synthetics/troubleshooting/_index.md

```diff
@@ -161,6 +164,32 @@ This might mean your private location is unable to reach the endpoint your API t
 
 This occurs when you attempt to mount a single file in a Windows-based container, which is not supported. For more information, see the [Docker mount volume documentation][16]. Ensure that the source of the bind mount is a local directory.
 
+### Restart the Worker Service without a reboot
```

Comment:
```suggestion
### Restart the Synthetics Private Location Worker service without a reboot
```

since this is a general troubleshooting page for synthetics, i think this context is necessary